frontend/src/components/dashboard/varianceHomePage/VarianceSearch.js

```diff
@@ -10,15 +10,7 @@ import CustomPropTypes from "@/customPropTypes";
  */
 const propTypes = {
   handleVarianceSearch: PropTypes.func.isRequired,
-  initialValues: PropTypes.shape({
-    region: PropTypes.arrayOf(PropTypes.string),
-    compliance_code: PropTypes.arrayOf(PropTypes.string),
-    major: PropTypes.string,
-    issue_date_after: PropTypes.string,
-    issue_date_before: PropTypes.string,
-    expiry_date_before: PropTypes.string,
-    expiry_date_after: PropTypes.string,
-  }),
+  initialValues: CustomPropTypes.varianceSearchInitialValues,
```

Comment:
That's probably not a propType worth storing, imo. But I haven't given it a ton of thought